src/site/content/en/blog/javascript-and-google-search-io-2019/index.md

```diff
@@ -0,0 +1,619 @@
+---
+title: JavaScript and Google Search
+subhead: Updates and insights from Google I/O 2019
+authors:
+  - martinsplitt
+  - ekharvey
+date: 2019-06-03
+hero: hero.png
+alt: A website with placeholder images instead of real cat images in the URL inspection tool within Search Console.
+description: |
+  We introduced a new Googlebot at Google I/O and took this opportunity to discuss improvements and best practices for JavaScript web apps in Google Search.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - seo
+  - discoverability
+  - javascript
+---
+
+## Meet the evergreen Googlebot
+
+This year Google I/O brought exciting news around Google Search and JavaScript: the much-awaited announcement of the [new evergreen Googlebot](https://webmasters.googleblog.com/2019/05/the-new-evergreen-googlebot.html).
+
+<figure class="w-figure">
+  <img src="evergreen-googlebot.png" alt="Googlebot holding the Chrome logo" style="max-width: 400px;">
+  <figcaption class="w-figcaption">
+    Googlebot is now running a modern Chromium rendering engine.
+  </figcaption>
+</figure>
+
+Googlebot now uses a modern Chromium engine to render websites for Google Search. On top of that, we test newer versions of Chromium to keep Googlebot updated in the future. We will update Googlebot within a few weeks of a stable Chrome release. This announcement is big news for web developers and SEOs, because it marks the arrival of [1000+ new features](https://caniuse.com/#compare=chrome+41,chrome+74), such as ES6+, IntersectionObserver, and Web Components v1 in Googlebot. 
+
+While the [session on new and upcoming features in Google Search](https://www.youtube.com/watch?v=ufcijo46LCU&list=PLKoqnv2vTMUPsSoDoVlUlgVkyh0OfjB-x) focused on new and upcoming features, we also had a session on JavaScript and Google Search. In this session we shared more details on improving JavaScript-driven web apps. We looked at:
+
+*   Feature detection and error handling
+*   Googlebot's pipeline for crawling, rendering and indexing
+*   Rendering strategies
+*   Testing tools for your website in Google Search
+*   Common challenges and possible solutions
+*   Best practices for SEO in JavaScript web apps
+
+Here's the full video:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/Ey0N1Ry0BPM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
```suggestion
<div style="width:100%; padding-top: 56.25%; position: relative;">
  <iframe style="width:100%; height: 100%;position: absolute; top: 50%; left: 50%; transform: translate(-50%,-50%);" src="https://www.youtube.com/embed/Ey0N1Ry0BPM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
</div>
```

This will center the video and make it responsive.